src/libraries/System.Net.Quic/tests/FunctionalTests/QuicListenerTests.cs

```diff
@@ -119,7 +119,7 @@ public async Task AcceptConnectionAsync_SlowOptionsCallback_TimesOut(bool useCan
 
             // Connect attempt should be stopped with "UserCanceled".
             var connectException = await Assert.ThrowsAsync<AuthenticationException>(async () => await connectTask);
-            Assert.Contains(TlsAlertMessage.UserCanceled.ToString(), connectException.Message);
+            Assert.Contains("UserCanceled", connectException.Message);
```

Comment:
What is the motivation for this change? What about `nameof`?

Reply:
It's an internal `enum` in security, it's not public. I removed all the shared source files, I haven't put any effort in figuring out why it was there. 
I see that the file with the enum doesn't have any dependencies, so it could be put back without dragging bunch of other stuff. I can do that.

Reply:
In the end, we're bringing back some code sharing so I reverted this change.